api/jobs-automation/alter_job.md

```diff
@@ -111,7 +41,95 @@ There is a distinction between runtime failures that do not cause the $JOB to cr
 In the event of a $JOB crash, the next start calculation follows the same formula,
 but it is always at least 5 minutes after the $JOB's last finish, to give an operator enough time to disable it before another crash.
 
-</Highlight>
 
 
-[add-policy]: /api/:currentVersion:/compression/add_compression_policy/#required-arguments
\ No newline at end of file
+## Samples
+
+- **Reschedule $JOB ID `1000` so that it runs every two days**:
+
+    ```sql
+    SELECT alter_job(1000, schedule_interval => INTERVAL '2 days');
+    ```
+
+- **Disable scheduling of the compression policy on the `conditions` hypertable**:
+
+    ```sql
+    SELECT alter_job(job_id, scheduled => false)
+    FROM timescaledb_information.jobs
+    WHERE proc_name = 'policy_compression' AND hypertable_name = 'conditions'
+    ```
+
+- **Reschedule continuous aggregate $JOB ID `1000` so that it next runs at 9:00:00 on 15 March, 2020**:
+
+    ```sql
+    SELECT alter_job(1000, next_start => '2020-03-15 09:00:00.0+00');
+    ```
+  
+- **Alter a columnstore_policy**:
+
+  You can pause and restart a columnstore policy, change how often the policy runs and the job scheduling.
```

Comment:
The samples part mentions both compression and columnstore policy - would be nice to stick to one.

Reply:
Thing is, that section is for columnstore policy. Annoying but correct.